_layouts/career.html

```diff
@@ -0,0 +1,66 @@
+<!Doctype html>
+<html id="careers">
+
+{% include head-header.html %}
+
+<section id="hero" class="light-text">
+</section>
+
+<section id="encyclopedia">
+    <div id="carsContent" class="noToc">
+
+        <button class="back" onClick="window.location.href='/careers/'"><img src="/images/careers/back_arrow.svg" alt="Back" style="vertical-align: middle; margin-right: 16px">Back to Careers</button>
+
+        <div id="vacancy">
+            <div class="head">
+                <p class="title">{{ page.position }}</p>
+                <div class="location">{{page.location}}<img src="/images/careers/location_icon.svg"></div>
+                <p class="date">{{page.dateString}}</p>
+            </div>
+
+            {{ content }}
+
+            <div id="mailBlock">
+                <p class="mailTitle">Send your CV on mail:</p><p class="mail">hr@thingsboard.io</p>
```

Comment:
When I see email - I want to click on it. Maybe you should add `<a href="mailto:...">` here?